hack/bootstrap-cluster.sh

```diff
@@ -1,8 +1,9 @@
 #!/bin/bash
 
 MODE=$1
+shift $#
 
-ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"/..
+ROOT="$(realpath --canonicalize-missing --quiet -- ${BASH_SOURCE[0]}/../..)"
```

Comment:
Now it's more complicated then original solution. I prefer original solution since it's oneliner and also realpath is behaving slightly differently on MacOS (not sure if also in this case). That `shift` is also slightly ugly (not much in this script, but other scripts are having ROOT as well, so it could be confusing).

Reply:
It's not more complicated, and actually It's much less complicated.

* one command, instead of three.
* no conditionals
* no output redirection
* one sub-shell instead of two.
* no dangling unquoted strings

Regardless, I've removed the long-args to significantly shorten the line length. That way line length isn't conflated with complexity, and I think it is much nicer on the eyes. ;-)

For MacOS, that's an interesting point. 
I'm not sure people are running the script on MacOS, but if they are then they should probably consider installing GNU coreutils prior. I'm told that's a common task for FOSS developers, and goes something like `brew install coreutils` (unconfirmed, not tested, just what I'm told)

Reply:
Ok with removal of swift it's better. LGTM, but please rebase first.